src/main/java/com/eviive/personalapi/controller/SkillController.java

```diff
@@ -156,7 +179,11 @@ public ResponseEntity<Void> sort(@RequestBody final List<SortUpdateDTO> sorts) {
         summary = "Delete a skill",
         responses = {
             @ApiResponse(responseCode = "204", description = "No Content"),
-            @ApiResponse(responseCode = "404", description = "Not Found")
+            @ApiResponse(
+                responseCode = "404",
+                description = "Not Found",
+                content = @Content(schema = @Schema(implementation = ProblemDetail.class))
+            )
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'annotation array initialization' child has incorrect indentation level 12, expected level should be 16.